src/lib/include/roc/address.h

```diff
@@ -85,6 +85,19 @@ typedef struct roc_address {
 ROC_API int
 roc_address_init(roc_address* address, roc_family family, const char* ip, int port);
 
+/** Set multicast interface address.
+ *
+ * @b Parameters
+ *  - @p address should point to a properly initialized address struct
+ *  - @p miface should point to a zero-terminated string with a valid IPv4 or
+ *       IPv6 multicast address
```

Comment:
...or network interface name?

Reply:
As discussed privately supporting interface name is not a good idea because:

* interface name can be mapped to the multiple addresses and who knows which one should be returned

* interface name can be mapped to the ivp4 or ipv6 versions of the same address and which version should be used is the next question

To make things simple we've decided to prohibit interface name usage.